extensions/wikia/ARecoveryEngine/js/InstartLogic/recovery.js

```diff
@@ -0,0 +1,36 @@
+define('ext.wikia.aRecoveryEngine.instartLogic.recovery', [
+	'ext.wikia.adEngine.adContext',
+	'wikia.log',
+	'wikia.window'
+], function (adContext, log, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.aRecoveryEngine.instartLogic.recovery',
+		context = adContext.getContext();
+
+	function isEnabled() {
+		var enabled = !!context.opts.instartLogicRecovery;
+
+		log(['isEnabled', enabled, log.levels.debug, logGroup]);
+		return enabled;
+	}
+
+	/**
+	 * When IL detects that user is blocking, it sets src=rec on page level params
+	 *
+	 * @returns {boolean}
+	 */
+	function isBlocking() {
+		var gptLib = win.googletag.pubads && win.googletag.pubads() && win.googletag.pubads().getTargeting('src'),
```

Comment:
Nit-picking: how about renaming the variable to `gptSrcKeyVal` in order to have it more descriptive?